build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
```

Comment:
A quick comment about why these two terse `bash`-isms are here would be helpful to me.